cernopendata/modules/fixtures/data/records/delphi-documentation-manuals.json

```diff
@@ -0,0 +1,205 @@
+[
+  {
+    "abstract": {
+      "description": "Detailed description of the DELPHI conditions database contents, and the tools coming with it."
+    },
+    "accelerator": "CERN-LEP",
+      "collaboration": {
+      "name": "DELPHI collaboration",
+      "recid": "27050"
+    },
+    "date_created": [
+      "1993"
+    ],
+    "date_published": "2024",
+    "distribution": {
+      "formats": [
+        "pdf"
+      ],
+      "number_files": 3
+    },
+    "experiment": [
+      "DELPHI"
+    ],
+    "files": [
+      {
+        "checksum": "adler32:18d0bebc",
+        "size": 404470,
+        "uri": "root://eospublic.cern.ch//eos/opendata/delphi/documentation/ddb/cargo600.pdf"
+      },
+      {
+        "checksum": "adler32:9ef43714",
+        "size": 229767,
+        "uri": "root://eospublic.cern.ch//eos/opendata/delphi/documentation/ddb/ddapp600.pdf"
+      },
+      {
+        "checksum": "adler32:63ea0076",
+        "size": 557435,
+        "uri": "root://eospublic.cern.ch//eos/opendata/delphi/documentation/ddb/dplot.pdf"
+      }
+    ],
+    "license": {
+      "attribution": "CC0"
+    },
+    "publisher": "CERN Open Data Portal",
+    "recid": "27001",
+    "title": "DELPHI database documentation",
+    "type": {
+      "primary": "Supplementaries",
+      "secondary": [
+        "DELPHI manuals"
```

Comment:
Let's call the `type.secondary` value simply "Manual", i.e. without leading DELPHI. This is because the facet values are usually experiment-agnostic, such as "Dataset > Collision" etc. The experiment-specific search filtering is provided by combining two facets, the type facet and the experiment facet. (Note that this is what we did for JADE logbooks for example, they have the value `Supplementaries.Logbook`.) In this way other experiments could add manuals more easily.

Reply:
fixed